python-main.js

```diff
@@ -1047,9 +1047,16 @@ function web_editor(config) {
         $('#editor').focus();
     }
 
-    function doConnect(e, serial) {
-        var p = Promise.resolve();
+    function showDisconnectError(event) {
+        var error = {"message": config["translate"]["webusb"]["err"]["device-disconnected"]};
+        webusbErrorHandler(error);
+    }
 
+    function doConnect(serial) {
+        // Show error on WebUSB Disconnect Events
+        navigator.usb.addEventListener('disconnect', showDisconnectError);
```

Comment:
Moved this from the bottom of `doConnect` to here, mostly because having code after the return statement can cause trouble.

I've also moved the anonymous function to a normal function, otherwise if we connected 3 times and then counter a disconnect error, then there would be 3 anonymous functions all trying to execute the same.